core/metamodel/src/main/java/org/apache/isis/core/runtime/threadpool/ThreadPoolSupport.java

```diff
@@ -124,16 +122,6 @@ public void close() throws Exception {
         return invokeAll(concurrentExecutor, callables);
     }
 
-    /**
-     * Executes specified {@code callables} on the default executor.
-     * See {@link ThreadPoolExecutor#invokeAll(java.util.Collection)}
-     * @param callables nullable
-     * @return non-null
-     */
-    public List<Future<Object>> invokeAll(final Callable<Object>... callables) {
```

Comment:
compiler warning: varargs possible heap pollution, so maybe just keep the API clean